include/oneapi/dpl/pstl/parallel_backend_utils.h

```diff
@@ -301,6 +306,79 @@ __set_symmetric_difference_construct(_ForwardIterator1 __first1, _ForwardIterato
     return __cc_range(__first2, __last2, __result);
 }
 
+namespace __detail
+{
+
+template <template <typename...> typename _Concrete, typename _ValueType, typename... _Args>
```

Comment:
```suggestion
template <template <typename, typename...> typename _Concrete, typename _ValueType, typename... _Args>
```
Optional suggestion, but would this be a better match to `template <typename _ValueType, typename... _Args>`?

Reply:
I believe it is not technically necessary. Even class templates that do not have variadic parameter packs at all, only at least one template parameter, would be suitable derived classes. That's how the variadic templates work.

But I would not object either, if that simplifies understanding for somebody.

Reply:
Yeah, either way is technically correct.  Your proposed method is more specific / restrictive and perhaps more simple to understand, so why not,  I'll make the change.

Reply:
Just to clarify, it's not even more restrictive :) It says that at least one template parameter is required - but that was already a requirement before, due to the definition of `derived_type` that passed `_ValueType` as the argument.

Reply:
Fair enough, in combination with the other parts of the definition, I agree.